module/motion/ScriptEngine/src/ScriptEngine.cpp

```diff
@@ -34,13 +34,12 @@ namespace module::motion {
     using message::behaviour::ServoCommand;
 
 
-    ScriptEngine::ScriptEngine(std::unique_ptr<NUClear::Environment> environment)
-        : Reactor(std::move(environment)), scripts() {
+    ScriptEngine::ScriptEngine(std::unique_ptr<NUClear::Environment> environment) : Reactor(std::move(environment)) {
 
         on<Script>("").then([this](const Script& script) {
             // Add this script to our list of scripts
             try {
-                scripts.insert(std::make_pair(utility::file::pathSplit(script.fileName).second, std::move(script)));
+                scripts.insert(std::make_pair(utility::file::pathSplit(script.fileName).second, script));
```

Comment:
Since c++17 its very rare that you need `std::make_pair` over `std::pair::pair` fyi.